src/main/java/org/folio/circulation/domain/ExpiredHoldsContext.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.circulation.domain;
+
+import org.folio.circulation.support.Result;
+
+import java.util.List;
+
+public class ExpiredHoldsContext {
+  private final int currPageNumber;
+  private final List<Result<MultipleRecords<Item>>> itemIds;
```

Comment:
I see item collection here. Why the variable called itemIds?

Reply:
fix